token.go

```diff
@@ -5,218 +5,262 @@
 package gocql
 
 import (
-	"bytes"
-	"crypto/md5"
-	"fmt"
-	"math/big"
-	"sort"
-	"strconv"
-	"strings"
-
-	"github.com/gocql/gocql/internal/murmur"
+    "bytes"
+    "crypto/md5"
+    "fmt"
+    "math/big"
+    "sort"
+    "strconv"
+    "strings"
+
+    "github.com/gocql/gocql/internal/murmur"
 )
 
 // a token partitioner
 type partitioner interface {
-	Name() string
-	Hash([]byte) token
-	ParseString(string) token
+    Name() string
+    Hash([]byte) Token
+    ParseString(string) Token
 }
 
-// a token
-type token interface {
-	fmt.Stringer
-	Less(token) bool
+// Token is a token
+type Token interface {
+    fmt.Stringer
+    Less(Token) bool
+}
+
+//TokenRange represents a token range.
+//The start token is exclusive and the end token is inclusive.
+type TokenRange struct {
```

Comment:
Do we need `TokenRange`? It seems to me that handling the sorted slice of tokens should be straightforward without needing to duplicate the start/end tokens in TokenRange. I'd rather avoid exporting another type if not necessary.

Reply:
No, we don't. Removed.